scripts/apps/search/directives/SearchPanel.js

```diff
@@ -306,6 +306,11 @@ export function SearchPanel($location, desks, privileges, tags, asset, metadata,
                         _.find(scope.tags.selectedFacets[type], (facet) => facet.value === key);
                 }
 
+                if (type === 'credit') {
+                    return scope.tags && scope.tags.selectedFacets[type] &&
```

Comment:
I would rewrite this function as:
```js
scope.hasFilter = function(type, key) {
    if (!scope.tags || !scope.tags.selectedFacets[type]) {
        return false;
    }

    const facet = scope.tags.selectedFacets[type];
    const isDesk = type === 'desk' && _.find(facet, (f) => f.value === key);
    const isCredit = type === 'credit' && facet.indexOf(scope.aggregations.credit[key].qcode) >= 0;
    
    return isDesk || isCredit || facet.indexOf(key) >= 0;
}
```
What do you think? You can keep whichever one you prefer, but I would remove the duplicate condition as suggested in the above comment either way.